layouts/partials/styles.html

```diff
@@ -0,0 +1,7 @@
+<link rel="stylesheet" href="https://unpkg.com/swiper/swiper-bundle.min.css" />
+
+{{ range (readDir "/assets/css") }}
```

Comment:
If desired, these assets can be bundled + minified for production

Reply:
I put in the minify command, but I'd rather keep it unbundled even for production so I can look at individual css files on the live site for debugging purposes, and to save an extra command (or lines). This isn't a strong preference though.

Also the css files are a drop in the bucket compared to all the other image and video files we'll be serving.

Reply:
Ok, just thought I would mention it.

I think historically reducing HTTP requests was considered good practice. Although, Google mentions in [their performance docs](https://developers.google.com/web/fundamentals/performance/get-started/httprequests-5#a_caveat) that this may not be the case for HTTP/2 requests. No need to try and prematurely optimize.